### PR TITLE
Hide extra options from various nodes if they're not enabled

### PR DESCRIPTION
**Pull request by YeldhamDev**

### CODE REVIEW

**Comment by bojidar-bg** [comment]
Wouldn't it be better to use `PROPERTY_USAGE_NOEDITOR` so that the properties get stored even when disabled?

**Comment by YeldhamDev** [comment]
@bojidar-bg Done!

**Review by akien-mga — APPROVED** [review]
Seems OK to me and reduz approved too on the dev chat. Might be worth a rebase to make sure it still works fine in 2021?

File: scene/resources/camera_effects.cpp [hunk 1]
```diff
@@ -129,6 +135,7 @@ bool CameraEffects::is_override_exposure_enabled() const {
 void CameraEffects::set_override_exposure(float p_exposure) {
 	override_exposure = p_exposure;
 	_update_override_exposure();
+	notify_property_list_changed();
```
Comment (akien-mga) [inline scene/resources/camera_effects.cpp:138]: I wonder if we couldn't call `notify_property_list_changed()` from `_validate_property` instead, only whenever something *should* change?

Comment (akien-mga) [reply scene/resources/camera_effects.cpp:138]: Or maybe more like, why is it called here for the float property? Isn't it sufficient to update the inspector only when boolean "enabled" properties are toggled?

Comment (YeldhamDev) [reply scene/resources/camera_effects.cpp]: Oops! These are not supposed to be there, sorry for that!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**